descriptors/ModuleDescriptor-template.json

```diff
@@ -90,6 +90,22 @@
         }
       ]
     },
+    {
+      "id": "declare-item-lost",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/circulation/loans/{id}/declare-item-lost",
```

Comment:
PUT is used to replace a resource representation. To me, this endpoint is describing a process, and so is better suited to using the POST method.

Reply:
Done